docs/guides/player-workflows.md

```diff
@@ -0,0 +1,58 @@
+# Player Workflows
+
+This document outlines many considerations for using Video.js for advanced player workflows. Be sure to read [the setup guide](setup.md) first!
+
+## Removing Players
+
+No matter the term used for it, web applications are becoming common. Not everything is a static, load-once-and-done web page anymore! This means that developers need to be able to manage the full lifecycle of a video player - from creation to destruction. Video.js supports player removal through the `dispose()` method.
+
+### [`dispose()`](http://docs.videojs.com/docs/api/player.html#Methodsdispose)
+
+This method is available on all Video.js players and [components](http://docs.videojs.com/docs/api/component.html#Methodsdispose). It is _the only_ supported method of removing a Video.js player from both the DOM and memory. For example, the following code sets up a player and then disposes it when media playback is complete:
+
+```js
+var player = videojs('my-player');
+
+player.on('ended', function() {
+  this.dispose();
+});
+```
+
+Calling `dispose()` will have a few effects:
+
+1. Trigger a `"dispose"` event on the player, allowing for any custom cleanup tasks that need to be run by your integration.
+1. Remove all event listeners from the player.
+1. Remove the player's DOM element(s).
+
+Additionally, these actions are recursively applied to _all_ the player's child components.
+
+> **Note**: Do _not_ remove players via standard DOM removal methods: this will leave listeners and other objects in memory that you might not be able to clean up!
+
+### Signs of an Undisposed Player
```

Comment:
The vdata error is a similar thing where the element was removed from the DOM but an event triggered for it.